### ISSUE TITLE
Is there configuration settings able to use existing mongodb

### ISSUE THREAD

**Issue by Shuliyey** [MLstate/PEPS#33]
Hi

I noticed PEPS uses mongodb, I've actually already have a mongodb setup on the host. is there configuration to use the existing mongodb

**Comment by hbbio** [comment]
Yes, look at the peps.sh launch script in peps/, you will see how Mongo parameters are passed.

**Comment by Shuliyey** [comment]
ty so much , will give it a try :smile:
